configure.ac

```diff
@@ -5,7 +5,7 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_INIT([Squid Web Proxy],[5.0.0-BZR],[http://bugs.squid-cache.org/],[squid])
+AC_INIT([Squid Web Proxy],[5.0.0-VCS],[http://bugs.squid-cache.org/],[squid])
```

Comment:
I doubt that calling master code v5.0.0 is a good idea. Yes, it solves some problems, but it also creates others.

However, I am not going to make that disagreement a blocking issue or waste your time discussing pros and cons. If you insist on assigning a numeric version to the master code in maintenance scripts and configure.ac, I can live with that as long as we have a proper master branch (which we now do).